js/package.json

```diff
@@ -69,7 +73,7 @@
     "react-dom": "^16.13.1"
   },
   "dependencies": {
-    "@aics/web-3d-viewer": "^2.3.1",
+    "@aics/web-3d-viewer": "^2.3.3-dev.0",
```

Comment:
if we are convinced this is good, let's not merge to nbvv until we make a true release build